en/components/grids_templates/sorting.md

```diff
@@ -117,9 +117,27 @@ public ngOnInit() {
 
 <div class="divider--half"></div>
 
-@@if (igxName === 'IgxGrid') {
+@@if (igxName === 'IgxGrid' || igxName === 'IgxTreeGrid') {
```

Comment:
We should add a separate sample for remote sorting on a TreeGrid as part of #1626.

Reply:
Addressed.